ceph-releases/luminous/ubuntu/16.04/daemon/disk_introspection.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+HOSTNAME=$(uname -n)
+MY_NAMESPACE=$(kubectl get pods --all-namespaces -o jsonpath="{.items[?(@.metadata.name == \"${HOSTNAME}\")].metadata.namespace}")
```

Comment:
This works:

`kubectl get pods --all-namespaces -o jsonpath="{.items[?(@.metadata.name == \"${H}\")].metadata.namespace}"`